### ISSUE TITLE
⛏️ Write a test to check whether we can create/update an object with negative price/amount/value

### ISSUE THREAD

**Issue by aktoboy** [akto-api-security/akto#179]
💭 Introduction:

🎯 Requirements:

✅ Task summary:

🙋🏼‍♂️ Questions:
If you have questions, need any help, or just want to hang out, make sure to join us on our [Discord server](https://discord.com/invite/Wpc6xVME4s).

**Comment by bhavyastar** [comment]
Hey @Ankita28g, I would like to work on this issue!

**Comment by Ankita28g** [comment]
Awesome! Assigned to you @bhavyastar. Happy hackfesting 😃

**Comment by aktoboy** [comment]
Hey @bhavyastar, I have updated the issue description to provide more details. Feel free to reach out if you need any help 😃. Happy hackfesting!!!

**Comment by siddoinghisjob** [comment]
Hey I am new to programming though I learnt java I don't know testing. Can you point me to some resources to learn about yaml, testing etc.

**Comment by Ankita28g** [comment]
how about you pair program with @ayushaga14? 🔥

**Comment by siddoinghisjob** [comment]
How does that work? I am ready for any learning opportunity 🤞

**Comment by Ankita28g** [comment]
Hi @siddoinghisjob this one is already assigned to @bhavyastar. Can you pick something else? Also, to learn how tests work with Akto, the best way is to local deploy Akto and run tests. 😊

**Comment by Ankita28g** [comment]
Hi @bhavyastar let us know if you need any help with this?

**Comment by bhavyastar** [comment]
Yes, Can we do a meeting tomorrow or just a discord call? I have some queries.
cc: @aktoboy @Ankita28g @Jittojoyes98

**Comment by Ankita28g** [comment]
hi totally, ping us on Discord. @bhavyastar

**Comment by Ankita28g** [comment]
Hi @bhavyastar thanks for your submission in Hackfest. 🔥 We are reviewing your work. Do these two below:

Join this group on [discord](https://discord.com/channels/1070706429402562733/1095281477748543618) for discussions around prizes? 🚀 🏆
Please fill this [form](https://forms.gle/fLXPKAJ8xcK9FhVP7) your PR to be considered for prizes!

**Comment by OshaibBeg** [comment]
Hey, I would like to contribute to this issue.

**Comment by atharvamalji** [comment]
Is this issue still unassigned? If so I would like to get assigned to solve the issue. This is my first time participating in hactoberfest.

**Comment by avneesh-akto** [comment]
I've assigned it to you, @atharvamalji . Happy hacking! Feel free to join our [Discord](https://discord.com/channels/1070706429402562733/1095281477748543618) if you need assistance.
